### PR TITLE
Update LeftAlignIndels documentation

### PR DESCRIPTION
**Pull request by nh13**
This caught me off-guard until I read the source code.

### CODE REVIEW

**Comment by ldgauthier** [comment]
Reads with multiple indels are skipped?

**Comment by droazen** [comment]
@ldgauthier That does appear to be the case based on this tool's `apply()` method:

```
        // move existing indels (for 1 indel reads only) to leftmost position within identical sequence
        int numBlocks = AlignmentUtils.getNumAlignmentBlocks(read);
        if ( numBlocks == 2 ) {
              ....
        }
        (otherwise do nothing)
```

**Comment by nh13** [comment]
@droazen I rolled my own tool here in case others find this issue: https://github.com/fulcrumgenomics/fgbio/pull/539

File: src/main/java/org/broadinstitute/hellbender/tools/LeftAlignIndels.java [hunk 1]
```diff
@@ -25,6 +25,8 @@
  * equivalent representations randomly or used different aligning conventions), so this tool can be used to left-align
  * them according to convention. </p>
  *
+ *<p><b>This tool will left-align reads with one and only indel</b>.</p>
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/LeftAlignIndels.java]: "one and only one"

**Pull request merged**